src/neural/network_random.cc

```diff
@@ -26,9 +26,9 @@ namespace lczero {
 
 class RandomNetworkComputation : public NetworkComputation {
  public:
-  RandomNetworkComputation(int delay) : delay_ms_(delay) {}
+  RandomNetworkComputation(int delay, int seed) : delay_ms_(delay), seed_(seed) {}
```

Comment:
Maybe give seed a default value of 0?

`RandomNetworkComputation(int delay, int seed = 0) : delay_ms_(delay), seed_(seed) {}`